server/grpc/server.go

```diff
@@ -14,13 +15,16 @@ import (
 	"github.com/cosmos/cosmos-sdk/server/config"
 	"github.com/cosmos/cosmos-sdk/server/grpc/gogoreflection"
 	reflection "github.com/cosmos/cosmos-sdk/server/grpc/reflection/v2alpha1"
-	"github.com/cosmos/cosmos-sdk/server/types"
 	_ "github.com/cosmos/cosmos-sdk/types/tx/amino" // Import amino.proto file for reflection
 )
 
+type grpcApp interface {
+	RegisterGRPCServer(gogogrpc.Server)
```

Comment:
Should be RegisterGRPCServerWithSkipCheckHeader it seems

Reply:
checking